src/components/AppNavigationIconBullet/AppNavigationIconBullet.vue

```diff
@@ -0,0 +1,95 @@
+<!--
+  - @copyright Copyright (c) 2019 John Molakvoæ <skjnldsv@protonmail.com>
+  -
+  - @author John Molakvoæ <skjnldsv@protonmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<docs>
+### Bullet
+This component is made to be used inside of the [AppNavigationItem](#AppNavigationItem) component slot: `icon`.
+
+```
+<AppNavigationIconBullet slot="icon" color="0082c9" />
+<AppNavigationIconBullet slot="icon" color="#0082c9" />
+
+```
+
+### AppNavigationItem example
+```
+<AppNavigationItem  title="Entry 2" :pinned="true">
+	<AppNavigationIconBullet slot="icon" color="0082c9" />
+</AppNavigationItem>
+```
+</docs>
+
+<template>
+	<div class="app-navigation-entry__icon--bullet" @click="onClick">
```

Comment:
```suggestion
	<div class="app-navigation-entry__icon-bullet" @click="onClick">
```
This is not a state :)